mkdocs.yml

```diff
@@ -20,7 +20,9 @@ nav:
   - Overview: index.md
   - Work with documentation: mkdocs.md
   - Jobs and tasks: states.md
-  - Simulator binaries: simulator_binaries.md
+  - Building and running app for Linux developers: for_linux_developers.md
```

Comment:
Is't the text too long to fit in left area ?

<img width="952" alt="Screenshot 2024-03-23 at 23 05 01" src="https://github.com/yaptide/yaptide/assets/7374733/c453153e-0069-4e8d-ba58-a627a75f2212">

See https://yaptide.github.io/yaptide/

Reply:
I shorten it to: 
- Building backend for developers
- Building backend for deployment using Docker

Should I shorten it more?